lib/kino/explorer.ex

```diff
@@ -17,6 +17,8 @@ defmodule Kino.Explorer do
 
   @type t :: Kino.JS.Live.t()
 
+  @numeric_types [{:f, 64}, :integer]
```

Comment:
I would write a function and I would already support some of the upcoming types:

```
defp numeric_type?({:s, _}), do: true
defp numeric_type?({:u, _}), do: true
defp numeric_type?({:f, _}), do: true
# For backwards compatibility
defp numeric_type?(other), do: other in [:float, :integer]
```